blueocean-pipeline-api/src/main/java/io/jenkins/blueocean/pipeline/creation/AbstractMultiBranchCreateRequest.java

```diff
@@ -0,0 +1,132 @@
+package io.jenkins.blueocean.pipeline.creation;
```

Comment:
Guess all these could be inside `api` package, like `io.jenkins.blueocean.pipeline.api`?